### PR TITLE
Added navbar UI main components

### PR DESCRIPTION
**Pull request by kabelic**

### CODE REVIEW

**Comment by kabelic** [comment]
Hello bro, i finished 'Setup the Navbar' ticket from trello.
please you can review my code, i'm open to your feedback

File: src/components/coinsComponent/index.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+import {CoinStyle} from '../../components/styles/Nav.styled';
 import './coinsComponent.css'
 
 const Coins = () => {
     return (
-        <div className='coinStyle'>
-            <Link to='/coin'>Coins</Link>
-        </div>
+        <CoinStyle>
+            <Link style={{textDecoration: 'none', color:'white'}} to='/coin'>Coins</Link>
```
Comment (yoloOnTheBattlefield) [inline src/components/coinsComponent/index.js:8]: don't use inline styles

File: src/components/coinsComponent/index.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
 import { Link } from 'react-router-dom';
+import {CoinStyle} from '../../components/styles/Nav.styled';
```
Comment (yoloOnTheBattlefield) [inline src/components/coinsComponent/index.js:2]: format your code

File: src/components/CurrencyComponent.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Currency, MoneyIcon, RightSide } from './styles/Nav.styled'
+import {RiArrowDropDownFill} from 'react-icons/ri'
```
Comment (yoloOnTheBattlefield) [inline src/components/CurrencyComponent.js:2]: format your code

File: src/App.js [hunk 4]
```diff
@@ -1,29 +1,66 @@
 import './App.css';
-import {createGlobal, ThemeProvider} from 'styled-components'
+import {createGlobalStyle, ThemeProvider} from 'styled-components'
```
Comment (yoloOnTheBattlefield) [inline src/App.js:2]: format your code

File: src/App.js [hunk 5]
```diff
@@ -1,29 +1,66 @@
...
 import {Route, Routes} from 'react-router-dom'
```
Comment (yoloOnTheBattlefield) [inline src/App.js:3]: format your code

File: src/App.js [hunk 6]
```diff
@@ -1,29 +1,66 @@
...
 import Coin from './pages/coin';
 import Portfolio from './pages/portfolio'
 import NavComponent from './components/navComponent/NavComponent';
+import React from 'react';
```
Comment (yoloOnTheBattlefield) [inline src/App.js:7]: move your abs imports at the top

File: src/App.js [hunk 7]
```diff
@@ -1,29 +1,66 @@
...
+ }
+
+
+class App extends React.Component {
+  state = {
+    on: true,
+  }
+
+   handleClick = () => {
+    this.setState({on: !this.state.on})
```
Comment (yoloOnTheBattlefield) [inline src/App.js:40]: format your code

File: src/App.js [hunk 8]
```diff
@@ -1,29 +1,66 @@
...
+        <>
+          <GlobalStyle/>
+          <Container>
+            <NavComponent/>
+              <Routes>
+                <Route path='/coin' element={<Coin/>}/>
+                <Route path='/portfolio' element={<Portfolio/>}/>
+              </Routes>
+          </Container>
+        </>
```
Comment (yoloOnTheBattlefield) [inline src/App.js:58]: no need for fragment

File: src/components/navComponent/NavComponent.js [hunk 9]
```diff
@@ -1,21 +1,26 @@
 import './NavComponent.css' 
 import Search from '../searchComponent';
 import Coins from '../coinsComponent';
+import CurrencyComponent from '../CurrencyComponent';
 import { Link } from 'react-router-dom';
+import { LeftSide, Nav, RightSide, RightCornerIcon, LinkStyle } from '../styles/Nav.styled';
+import {SiNintendoswitch} from 'react-icons/si'
```
Comment (yoloOnTheBattlefield) [inline src/components/navComponent/NavComponent.js:7]: format your code
and abs imports at the top

File: src/components/searchComponent/index.js [hunk 10]
```diff
@@ -1,11 +1,12 @@
 import './Search.css'
-
+import { SearchStyle, SearchIcon } from '../styles/Nav.styled';
+import {RiSearch2Line} from 'react-icons/ri'
```
Comment (yoloOnTheBattlefield) [inline src/components/searchComponent/index.js:3]: format your code
and move code at top

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

**Pull request merged**